DESCRIPTION

```diff
@@ -21,7 +21,8 @@ SystemRequirements: CmdStan <https://mc-stan.org/users/interfaces/cmdstan>
 Imports: 
     checkmate,
     processx,
-    R6
+    R6,
+    jsonlite
```

Comment:
I would say the minimum is 1.2, since that is when `write_json` was introduced (I am going to switch to that as opposed to toJSON with write. Will add that requirement.